Update NekoDesu.co.uk's Visual Style While Fixing a Couple of Things Along the Way

Onosume opened the pull request:
This range of commits makes quite a lot of changes to the webpage including:

- Moving to Bootstrap to simplify the css file and allowing for easier use of the typical column layout
- Make the style match nkd.su; same font, colours etc...
- Fixing the language changer bar so it's easier to see and use
- Have the Twitter link go the hashtag page rather than the search page so people don't see irrelevant tweets or users

In my opinion this looks a lot better than it did before and fits in better with nkd.su. The three main links can be switched around as is appropriate depending on how much prominence we want to give certain ones on mobile.

colons commented on style.css:
```diff
@@ -0,0 +1,82 @@
+}
+
+.h1 {
+    color: #e87c36 !important;
```
i've tried pretty hard to make orange mean clickable in _all_ cases on nkd.su, so i have to admit i am not a fan of this or the `.h2`

theshillito replied:
Beat me by seconds lol.

Onosume replied:
```suggestion
    color: white !important;
```
I completely forgot this was the design intention. A change to white should be suitable enough?

theshillito commented:
This looks great! I'm all for this update. The one concern I have is that, with nkd.su, anything that is orange is a link. On the new version, the big titles in the top third are orange.

colons commented:
this isn't new to this PR (it's true of the current site too), but im unsure why all of the headings are divs that are just styled to _look_ like headings; this makes navigating using screen readers harder than it needs to be

colons commented:
(sorry, i should also add that i am also a fan of almost everything else that you've done here, gw, love it)

Onosume commented:
> this isn't new to this PR (it's true of the current site too), but im unsure why all of the headings are divs that are just styled to _look_ like headings; this makes navigating using screen readers harder than it needs to be

That's a fair point and one that I don't have an answer to other than I copied it from the old version without thinking about it. Definitely something to look at.

But yeah thanks for the feedback both!

NekomimiScience commented:
If you're up for updating this pull request with the requested changes, I'm happy to merge it by the way. 👍

theshillito commented:
I'm happy for this to be merged ^_^

Onosume commented:
Finally got round to updating this and looking at colons' comments.

Only clickable links are now in orange.
Changed hx styled divs to hx tags.
Removed some divs that were containing the contents as they seemed ultimately unnecessary

theshillito commented:
Awesome. Thanks @Onosume @NekomimiScience and @colons - looks great
